corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -3,6 +3,14 @@
 
   <script src='https://api.mapbox.com/mapbox.js/v3.3.1/mapbox.js' async></script>
   <link href='https://api.mapbox.com/mapbox.js/v3.3.1/mapbox.css' rel='stylesheet' />
+  <script src="https://api.mapbox.com/mapbox-gl-js/v1.11.1/mapbox-gl.js" async></script>
```

Comment:
Because we've deployed yarn, should this now use a module https://github.com/mapbox/mapbox-gl-geocoder#usage-with-a-module-bundler